Jenkinsfile

```diff
@@ -340,6 +340,30 @@ def k8sTest(Map args = [:]) {
   }
 }
 
+/**
+* This method runs the GitHub action
+*/
+def githubAction(Map args = [:]) {
+  // We don't support branches/tags for the time being.
+  if (!isPR()) { return }
+
+  // Notify the status with a specific GitHub Check
+  withGithubNotify(context: args.context) {
+    // Provision a worker with the given labels. For the time being, let's provision a worker.
+    // We can revisit this to maybe use the async mode when triggering the github action.
+    withNode(labels: args.label, forceWorkspace: true){
+
+      // TODO: This is the section regarding how to run the GitHub actions.
```

Comment:
@nxei , this is the section to be implemented.

Reply:
Added code to this section.